intro-HPC/ch_introduction.tex

```diff
@@ -142,8 +185,8 @@ \section{What is the \hpcInfra?}
 with \emph{cpuset} support and \emph{BLCR} modules.
 \fi
 \ifleuven
-All the nodes in the \hpc run under the ``\operatingsystem'' operating system,
-version 6.
+All the nodes in the \hpc run under the ``\operatingsystem'' operating system.
+%version 6.
```

Comment:
\operatingsystem is defined already in macros.tex (leuven) with its version:
\macro{\operatingsystem}{Redhat Enterprise Linux 6.5 (Santiago)}
So I tried to avoid having the same info twice.

From: Kenneth Hoste [mailto:notifications@github.com]
Sent: donderdag 18 februari 2016 14:04
To: hpcugent/vsc_user_docs <vsc_user_docs@noreply.github.com>
Cc: Mag Selwa <mag.selwa@kuleuven.be>
Subject: Re: [vsc_user_docs] Feature/leuven chapter1 (#106)


In intro-HPC/ch_introduction.tex<https://github.com/hpcugent/vsc_user_docs/pull/106#discussion_r53310316>:

> @@ -142,8 +185,8 @@ \section{What is the \hpcInfra?}

>  with \emph{cpuset} support and \emph{BLCR} modules.

>  \fi

>  \ifleuven

> -All the nodes in the \hpc run under the ``\operatingsystem'' operating system,

> -version 6.

> +All the nodes in the \hpc run under the ``\operatingsystem'' operating system.

> +%version 6.

?

—
Reply to this email directly or view it on GitHub<https://github.com/hpcugent/vsc_user_docs/pull/106/files#r53310316>.